src/Microsoft.DotNet.Cli.Utils/StreamForwarder.cs

```diff
@@ -83,11 +83,6 @@ public void Read(TextReader reader)
 
         private void WriteBuilder()
         {
-            if (_builder.Length == 0)
```

Comment:
Isn't the real problem in the method directly above this one?

```C#
private static readonly char s_flushBuilderCharacter = '\n';

...

public void Read(TextReader reader)
{
...

    if (currentCharacter == s_flushBuilderCharacter)
    {
        WriteBuilder();
    }
    else if (! s_ignoreCharacters.Contains(currentCharacter))
    {
        _builder.Append(currentCharacter);
    }
```

The newlines aren't getting written to the builder.